WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/MigratableStateTracker.swift

```diff
@@ -14,14 +14,14 @@ struct MigratableStateTracker {
 
     /// Tracks an event representing the WordPress migratable state.
     /// If WordPress is not installed, nothing is tracked.
-    func track() {
-        let installationState = MigrationAppDetection.getWordPressInstallationState()
+    func track(_ installationState: WordPressInstallationState) {
         switch installationState {
         case .wordPressInstalledAndMigratable:
             self.trackMigratable()
         case .wordPressInstalledNotMigratable:
             self.trackNotMigratable()
-        default:
+        case .wordPressNotInstalled:
+            /// Not tracked.
             break
         }
     }
```

Comment:
I think this file is not needed anymore, rigth?

Reply:
Yep, this was a remnant of the release branch merge. Wiped in 566646717bb9f93d1a0f2419f7c20380afbc96ab.